frontend/src/metabase/admin/people/components/PeopleList.jsx

```diff
@@ -53,12 +54,12 @@ const mapDispatchToProps = {
     ),
 };
 
+const defaultUsersValue = [];
```

Comment:
@sloansparger Could/should this change be outside Metabot PR? I don't think the admin change/RTK Query refactoring doesn't directly relate to Metabot in the grand scheme of things.

I'll leave it to your discretion whether to leave it here.